### PR TITLE
Added video component and fixed failing tests

### PR DESCRIPTION
**Pull request by ngyongzhen**
**New**
- Added ik-video component
- Added tests for ik-video component

**Updates**
- Updated package.json to reflect new versions

**Fixes**
- Fix failing tests resulted from imagekit-javascript version upgrade

### CODE REVIEW

File: sdk/lib/src/imagekitio-angular/ik-video/ik-video.component.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  template: `<video controls><source src={{src}}>`,
+})
+
+export class IkVideoComponent implements OnInit {
+    @Input('urlEndpoint') urlEndpoint:string;
+    @Input('path') path:string; //required
+    @Input('src') src:string; //required
+    @Input('transformation') transformation:Array<Object> = [];
+    @Input('transformationPosition') transformationPosition:string;
+    @Input('queryParameters') queryParameters:Object;
```
Comment (amanforindia) [inline sdk/lib/src/imagekitio-angular/ik-video/ik-video.component.ts:15]: Possibly can be refined a bit like here: https://github.com/imagekit-developer/imagekit-react/blob/master/src/components/IKVideo/props.ts#L6

File: sdk/src/app/app.component.html [hunk 2]
```diff
@@ -1,5 +1,10 @@
 <div className="App">
   <h1>Hi! This is an ImageKit Angular SDK Demo!</h1>
+  <ik-video src="https://ik.imagekit.io/demo/sample-video.mp4"></ik-video>
+  <p></p>
+  <ik-video src="https://ik.imagekit.io/demo/sample-video.mp4" [transformation]="transformationOne"></ik-video>
+  <p></p>
+  <ik-video src="https://ik.imagekit.io/demo/sample-video.mp4" [transformation]="transformationTwo"></ik-video>
```
Comment (amanforindia) [inline sdk/src/app/app.component.html:7]: I think it is an object array, not a string. Maybe you can use an example from react?

Comment (ngyongzhen) [reply sdk/src/app/app.component.html:7]: Do you mean for transformation or src?

File: sdk/lib/src/imagekitio-angular/ik-video/ik-video.component.ts (hunk 1 above)
Comment (ngyongzhen) [reply sdk/lib/src/imagekitio-angular/ik-video/ik-video.component.ts:15]: The typescripting will be implemented in a separate PR

File: sdk/src/app/app.component.html (hunk 2 above)
Comment (amanforindia) [reply sdk/src/app/app.component.html:7]: transformation

**Pull request merged**